kolibri/core/assets/src/utils/errorReportUtils.js

```diff
@@ -6,13 +6,35 @@ class ErrorReport {
   getErrorReport() {
     throw new Error('getErrorReport() method must be implemented.');
   }
+  getDeviceInfo() {
+    return {
+      type: /Mobi|Android/i.test(navigator.userAgent) ? 'Mobile' : 'Desktop',
+      platform: navigator.platform,
+      screen: {
+        width: window.innerWidth,
+        height: window.innerHeight,
+      },
+    };
+  }
+  getBrowserInfo() {
+    return navigator.userAgent;
+  }
 }
 
 export class VueErrorReport extends ErrorReport {
+  constructor(e, vm) {
+    super(e);
+    this.vm = vm;
+  }
   getErrorReport() {
     return {
       error_message: this.e.message,
       traceback: this.e.stack,
+      context: {
+        component: this.vm.$options.name || this.vm.$options._componentTag || 'Unknown Component',
+        browser: this.getBrowserInfo(),
```

Comment:
We can use `browser` in [browserInfo.js](https://github.com/learningequality/kolibri/blob/b8aae9f6079627f49a0e1d0b89b68777aa0db8d4/kolibri/core/assets/src/utils/browserInfo.js)